### DISCUSSION TITLE
Keeping Buttons Pressed to loop over Range

### DISCUSSION THREAD

**Discussion by SimonSiebold** [forntoh/LcdMenu#269] [Q&A] [answered]
Hi,

I have created an ITEM_INT_RANGE with values ranging from 1 to 512. Currently, I need to press the button each time to increment or decrement the value by 1.

Is there a way to keep the button pressed to continuously cycle through the values? The button used includes a method like button.toggled().

Thank you for your support!

**Comment by kimbrussow** [comment]
Are you using a rotary encoder?

**Reply by SimonSiebold** [reply]
No, I have 4 simple Pushbuttons, Back, Enter, Up, Down

**Comment by TBAMax** [comment]
> Is there a way to keep the button pressed to continuously cycle through the values? 

You need a custom buttonadapter for that.

**Comment by forntoh** [comment, marked as answer]
Added in #370
